spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -283,7 +290,6 @@ def main(argv: Sequence[str]):
             # If output folder does not exist, create it
             if not (path_out == '' or os.path.exists(path_out)):
                 os.makedirs(path_out)
-
```

Comment:
This is just a nitpick, but ideally there would be no "empty diffs" like this, that way we only change the lines that are absolutely necessary for the PR.

(The reason for this is so that the "blame view" (see [here](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blame/b80ab5374ca6778d5f816672a67802ee2caa7cbb/spinalcordtoolbox/scripts/sct_deepseg.py#L241)) remains clean, so that we can properly identify the history of each line of code.)

Reply:
Oh I did this because this was getting caught in the `code quality` check. And it passed after the removal😅

Reply:
I think what happened is that... before, there was 1 empty line, and the code quality check found 2 empty lines, but the "fix" was 0 empty lines. :sweat_smile: 

Keeping the same amount of empty lines should make the "diff" go away. :blush: